app/src/utils/utils.ts

```diff
@@ -28,22 +42,69 @@ export function isValidAddress(val: string | undefined) {
   return val && isAddress(val);
 }
 
-// Expects a unix timestamp and will return a human readable message of how far in the past/future it is
-export function daysAgo(val = 0) {
-  // Use a formatter to establish "in 10 days" vs "10 days ago"
-  const formatter = new Intl.RelativeTimeFormat();
-  // Number of days since now
-  const deltaDays = (val * 1000 - Date.now()) / (1000 * 3600 * 24);
+// --- Grants + Cart ---
+const CART_KEY = 'cart';
+const DEFAULT_CONTRIBUTION_TOKEN_ADDRESS = '0x6B175474E89094C44Da98b954EedeAC495271d0F'; // DAI
+const DEFAULT_CONTRIBUTION_AMOUNT = 5; // this is converted to a parsed BigNumber at checkout
```

Comment:
would it make sense having a constant files and dumping all these there ?

Reply:
The reason I put them here instead of the existing `app/src/utils/constants.ts` file is that these don't ever need to be used outside this file, so we can just define them here for simplicity/readability. Open to being convinced otherwise though!